grails-app/controllers/org/pih/warehouse/user/DashboardController.groovy

```diff
@@ -198,9 +198,16 @@ class DashboardController {
             // Save the warehouse selection for "last logged into" information
             if (user) {
                 //userInstance.rememberLastLocation = Boolean.valueOf(params.rememberLastLocation)
-                user.lastLoginDate = new Date()
-                user.save(flush: true)
-                session.user = user
+
+                /*
+                 * Band-aid for OBGM-355, to prevent the following exception at dashboard launch:
+                 * javax.persistence.TransactionRequiredException: no transaction is in progress
+                 */
+                user.withTransaction {
+                    user.lastLoginDate = new Date()
+                    user.save(flush: true)
+                    session.user = user
+                }
```

Comment:
@jmiranda @awalkowiak I wanted to highlight this band-aid -- let me know if there's a more performant way to avoid `TransactionRequiredException` than by wrapping the save call in a `withTransaction` block